web/modules/custom/bnf/bnf_client/src/Form/BnfImportConfirmForm.php

```diff
@@ -45,9 +46,11 @@ public function __construct(
     protected LoggerInterface $logger,
     ConfigFactoryInterface $configFactory,
     EntityTypeManagerInterface $entityTypeManager,
+    TranslationInterface $stringTranslation,
   ) {
     $this->baseUrl = $configFactory->get(SettingsForm::CONFIG_NAME)->get('base_url');
     $this->nodeStorage = $entityTypeManager->getStorage('node');
+    $this->setStringTranslation($stringTranslation);
```

Comment:
Why is this necessary? You dont seem to do anything in relation to translations in the rest of the file?

Reply:
Well, this should have been a seperate fixup commit.

It is the proper way to do dependency injection with the traits. If you don't do this, `StringTranslationTrait` will fall back to getting the service using `\Drupal`, which defeats the purpose of DI.